assets/airports/ksfo.json

```diff
@@ -127,27 +127,27 @@
     {
       "name":        ["28R", "10L"],
       "name_offset": [[0, 0.3], [0, 0.3]],
-      "end":    [ ["N37.6135324", "W122.3571410"], ["N37.6287386", "W122.3933911"]],
+      "end":    [ ["N37.6135324", "W122.3571410", "13.0ft"], ["N37.6287386", "W122.3933911", "5.5ft"]],
```

Comment:
@tedrek Just curious... where are you getting these to-the-inch elevation values? A DEM model maybe? Seems super high precision.